src/attributes.jl

```diff
@@ -965,6 +965,8 @@ struct VariableName <: AbstractVariableAttribute end
     VariablePrimalStart()
 
 A variable attribute for the initial assignment to some primal variable's value that the optimizer may use to warm-start the solve.
+
+The value `nothing` means the solver is free to decide which value to use. This attribute can be set back to `nothing` after being set to a numerical value.
```

Comment:
The `VariablePrimalStart` is a hint that the optimizer "may" use to warm-start the solve when it's set. It's redundant to say the solver "is free to decide which value to use" when it's set to `nothing`, because this is also true when it's set to a number. Just say "May be a number or `nothing` (unset)."

Reply:
Done.